src/coreComponents/constitutive/fluid/PVTFunctions/PVTFunctionHelpers.hpp

```diff
@@ -99,10 +99,17 @@ struct PVTFunctionHelpers
       real64 const PEnd = stod( inputParameters[3] );
       real64 const dP = stod( inputParameters[4] );
 
-      real64 const TStart = stod( inputParameters[5] );
-      real64 const TEnd = stod( inputParameters[6] );
+      constexpr real64 T_K = 273.15;
+
+      real64 const TStart = stod( inputParameters[5] ) - T_K;
+      real64 const TEnd = stod( inputParameters[6] ) - T_K;
       real64 const dT = stod( inputParameters[7] );
 
+      GEOSX_THROW_IF( TStart < 0, "Temperature must be in Kelvin and must be positive",
+                      InputError );
+      GEOSX_THROW_IF( TEnd < 0, "Temperature must be in Kelvin and must be positive",
+                      InputError );
```

Comment:
You are not consistent with `The temperature must be between 283.15 and 623.15 Kelvin.`. Is it OK?

Reply:
Yes you are right, I added a check to make sure that the temperature are within the range expected for the CO2-brine model, and we will update later if other models are added.